pkg/controller/directimagemigration/task.go

```diff
@@ -207,6 +209,15 @@ func (t *Task) Run() error {
 
 // Advance the task to the next phase.
 func (t *Task) next() error {
+	// Write time taken to complete phase
+	t.Owner.Status.StageCondition(migapi.Running)
+	cond := t.Owner.Status.FindCondition(migapi.Running)
+	if cond != nil {
+		now := time.Now().UTC()
+		elapsed := now.Sub(cond.LastTransitionTime.Time.UTC())
```

Comment:
Minor NIT: you can probably replace this with
```
elapsed := time.Since(cond.LastTransitionTime.Time)
```

From what I have read, this pattern is known to produce fewer bugs. I have been guilty of not using it as well though. Also, I am not sure how to handle UTC but just spraying thoughts if you want to play with it.

Reply:
ACK. I saw DVM controller was using UTC in its elapsed time calculation and just copied that assuming there was a reason for the UTC conversion.

Reply:
Changed